pkg/junit/junit.go

```diff
@@ -0,0 +1,71 @@
+// Package junit describes the kubernetes/test-infra definition of "junit", and provides
```

Comment:
This needs a header?

Also, if we break out `prow/` into a repo, this will need to come with?